packages/wrangler/package.json

```diff
@@ -122,7 +124,8 @@
       "node_modules/(?!find-up|locate-path|p-locate|p-limit|yocto-queue|path-exists|execa|strip-final-newline|npm-run-path|path-key|onetime|mimic-fn|human-signals|is-stream|get-port)"
     ],
     "moduleNameMapper": {
-      "clipboardy": "<rootDir>/src/__tests__/helpers/clipboardy-mock.js"
+      "clipboardy": "<rootDir>/src/__tests__/helpers/clipboardy-mock.js",
+      "miniflare/cli": "<rootDir>/../../node_modules/miniflare/dist/src/cli.js"
```

Comment:
Out of curiosity, why is this needed? Is it because of calls like:

```js
require.resolve("miniflare/cli")
```

Reply:
That's exactly it. `jest` hijacks the module loader so it can do stuff like mocks, but looks like it struggles with this specific call (because it's an es import? maybe, dunno.)